app/src/androidTest/java/ch/epfl/sdp/peakar/points/ComputePOIPointsTest.java

```diff
@@ -46,11 +46,13 @@
     private static Context mContext;
 
     @BeforeClass
-    public static void setUserPoint(){
+    public static void setUserPoint() throws InterruptedException {
         //Set userPoint to the mont blanc
         mContext = ApplicationProvider.getApplicationContext();
         userPoint = UserPoint.getInstance(mContext); 
         userPoint.setLocation(45.802537, 6.850328, 4809, 0);
+        //new ComputePOIPoints(mContext);
+        //Thread.sleep(20000);
```

Comment:
Should this be removed?

Reply:
No, as it will these comments will be removed after the test I think, so we might keep it there